README.md

```diff
@@ -1,24 +1,143 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
```

Comment:
On 3:51 of the video I say that what I like the most, or what Im proud of is the design and how close to the original it was.
the requirements say "you should also highlight some interesting piece of code or something you built that you are very proud of."
I highlighted the design I built instead of code, like described.

Reply:
- [ ] Dear @redmarverick, I see that one, I was expecting to see some codebase implementation, hence a `piece of code`. What you highlight is a feature and its good. but still we are interested in the code.